### ISSUE TITLE
[MODULE] - Password Strength/complexity 

### ISSUE THREAD

**Issue by h3lmii** [code-kern-ai/bricks#286]
**Please describe the module you would like to add to bricks**
Given a password,I want to test its complexity
**Do you already have an implementation?**
If so, please share it here. For instance:
I found a math-based approach: https://www.linkedin.com/pulse/math-based-approach-password-strength-walt-powell-cissp-cism/
**Additional context**

**Comment by LeonardPuettmannKern** [comment]
That's a really cool idea @h3lmii ! I think this should also be relatively easy to implement. Here they use regex for this: https://stackoverflow.com/questions/16709638/checking-the-strength-of-a-password-how-to-check-conditions

**Comment by h3lmii** [comment]
@LeonardPuettmannKern yes you're right.I've tested the Regex method, it's easier and efficient.
